custom_model_runner/datarobot_drum/resource/components/Python/generic_predictor/generic_predictor.py

```diff
@@ -105,7 +106,13 @@ def _materialize(self, parent_data_objs, user_data):
             binary_data, mimetype = StructuredInputReadUtils.read_structured_input_file_as_binary(
                 input_filename
             )
-            transformed_df = self._predictor.transform(binary_data=binary_data, mimetype=mimetype)
+            transformed_output = self._predictor.transform(
+                binary_data=binary_data, mimetype=mimetype
+            )
+            if isinstance(transformed_output, tuple):
```

Comment:
So just once again: do you agree that `transformed_output` is always a tuple? Why then we check for DataFrame type?

Reply:
@yakov-g finally address this and agree output is always a tuple and eliminated hanlding of dataframe.